rindti/models/base_model.py

```diff
@@ -26,7 +26,6 @@ class BaseModel(LightningModule):
     def __init__(self, **kwargs):
         super().__init__()
         self.save_hyperparameters()
-        return kwargs["model"]
```

Comment:
I did this because we want to save **all** hyperparameters in every instance of running the model, but we don't need **all** the stuff during model instantiation, so I moved this functionality to BaseModel

Reply:
Yeah, I just don't see the point of returning the models name here

Reply:
I see you idea behind returning this part of the args, smart idea. But I think `__init__` is not meant to be used like this ^^

Reply:
it's not the model's name here, it's the kwargs of the model. But I guess it makes sense to actually use self.hparams from this point on, since we don't really need kwargs after that?

Reply:
I removed the return from the model comletely. In the testing (and pretraining), I reset `kwargs=kwargs["mode"]`